src/routes/zoom.routes.js

```diff
@@ -0,0 +1,217 @@
+const { Router } = require('express')
+const got = require('got')
+const mongoose = require('mongoose');
+
+const Usuario = require('../models/usuario')
+const ZoomDatosUsuarios = require('../models/zoomDatosUsuarios')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Backend/pullRequest?prid=7157741)
```suggestion
const ZoomDatosUsuarios = require("../models/zoomDatosUsuarios")
```